azure/functions/decorators/function_app.py

```diff
@@ -343,6 +343,8 @@ def function_name(self, name: str,
         additional setting fields
         :return: Decorator function.
         """
+        if setting_extra_fields is None:
+            setting_extra_fields = {}
```

Comment:
Is this needed? Or does something break when settings is none?

Reply:
Previously we were initializing setting_extra_fields to an empty dict. This is keeping the same format so that if something is dependent on setting_extra_fields being {} then it's not broken